src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,86 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './NewCommentForm.scss';
+import { postComment } from '../../api/comments';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
-    >
-      Add a comment
-    </button>
-  </form>
-);
+type UpdateComments = () => void;
+
+type Props = {
+  postId: number;
+  updateComments: UpdateComments;
+};
+
+export const NewCommentForm: React.FC<Props> = (props) => {
+  const { postId, updateComments } = props;
+  const [nameForComment, setNameForComment] = useState<string>('');
+  const [emailForComment, setEmailForComment] = useState<string>('');
```

Comment:
types are unnecessary for primitive
```suggestion
  const [nameForComment, setNameForComment] = useState('');
  const [emailForComment, setEmailForComment] = useState('');
```